packages/vue-urql/src/useQuery.test.ts

```diff
@@ -148,23 +363,86 @@ describe('useQuery', () => {
         () => subject.source as OperationResultSource<OperationResult>
       );
 
-    const pause = ref(true);
-
-    const _query = useQuery({
+    const query = useQuery({
       query: `{ test }`,
-      pause: () => pause.value,
+      pause: true,
     });
-    const query = reactive(_query);
 
     expect(executeQuery).not.toHaveBeenCalled();
 
-    pause.value = false;
+    query.resume();
     await nextTick();
-    expect(query.fetching).toBe(true);
+    expect(query.fetching.value).toBe(true);
 
     subject.next({ data: { test: true } });
 
-    expect(query.fetching).toBe(false);
-    expect(query.data).toEqual({ test: true });
+    expect(query.fetching.value).toBe(false);
+    expect(query.data.value).toHaveProperty('test', true);
+  });
+
+  it('pauses query with ref variable', async () => {
```

Comment:
looks like this resolves https://github.com/urql-graphql/urql/issues/3633

Reply:
it is